src/base/object_representation_sr.c

```diff
@@ -4283,6 +4305,20 @@ or_mvcc_get_header (RECDES * record, MVCC_REC_HEADER * mvcc_header)
 
 exit_on_error:
   return (rc == NO_ERROR && (rc = er_errid ()) == NO_ERROR) ? ER_FAILED : rc;
+
+underflow_detected:
+  mvcc_header->repid = 0;
+  mvcc_header->mvcc_flag = 0;
+  mvcc_header->chn = NULL_CHN;
+  mvcc_header->mvcc_ins_id = MVCCID_NULL;
+  mvcc_header->mvcc_del_id = MVCCID_NULL;
+
+  if (rc != NO_ERROR)
+    {
+      goto exit_on_error;
+    }
+
+  return NO_ERROR;
```

Comment:
The `rc != NO_ERROR)` seems to be always true because or_underflow() returns ER_TF_BUFFER_UNDERFLOW. Then, we can just move this `underflow_detected` section above exit_on_error without this.
```suggestion 
 exit_on_error:
   return (rc == NO_ERROR && (rc = er_errid ()) == NO_ERROR) ? ER_FAILED : rc;
```

Reply:
Forget this, this was on the revert code.